source/concepts/what-is-ember-js.md

```diff
@@ -124,15 +124,18 @@ You can tell Ember about these dependencies like this:
 
 ```javascript
 var president = Ember.Object.create({
-  firstName: "Barack",
-  lastName: "Obama",
+  init: function() {
+    this._super.apply(this, arguments);
+    this.set('firstName', 'Barack');
```

Comment:
arguments is "deprecated" in ES2015, Babel handles this fine:

```js
function bob(...args) {
  return this.fmt(...args);
}
```
```js
function bob() {
  return this.fmt.apply(this, arguments);
}
```

More in depth explanation why I put that in quotes: https://esdiscuss.org/notes/2015-03-25